sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -125,8 +129,14 @@ object JdbcUtils extends Logging {
       dialect: JdbcDialect): Iterator[Byte] = {
     val conn = getConnection()
     var committed = false
+    val supportsTransactions = Try( 
```

Comment:
Hi Sean, hi Reynold,
thanks for your comments!
I have added the blank, changed the title, but I am not sure what to 
change this code to.
The idea is to swallow any exception. My intention is to make code 
tolerant against flaky drivers and not risk the entire functionality in 
case some driver did not properly implement some tiny aspect (like 
transaction capabilties).
Can you please advice on how this should be handled? - I could also 
remove the entire try-catch. Just thought this may give more stability.
Thanks,
Christian

On 25.11.2015 19:56, Reynold Xin wrote:
>
> In 
> sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala 
> <https://github.com/apache/spark/pull/9973#discussion_r45905487>:
>
> > @@ -125,8 +129,14 @@ object JdbcUtils extends Logging {
> >        dialect: JdbcDialect): Iterator[Byte] = {
> >      val conn = getConnection()
> >      var committed = false
> > +    val supportsTransactions = Try(
>
> Sounds good!
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/apache/spark/pull/9973/files#r45905487>.
>